Userland/Libraries/LibJS/Runtime/DatePrototype.cpp

```diff
@@ -101,6 +102,11 @@ JS_DEFINE_NATIVE_FUNCTION(DatePrototype::get_date)
     auto* this_object = typed_this(vm, global_object);
     if (!this_object)
         return {};
+
+    if (this_object->is_invalid()) {
+        return Value(NAN);
+    }
+
```

Comment:
- Please use the `js_nan()` function instead of `Value(NAN)`
- Please omit the curly braces for an if-body with only one line

(everywhere in this PR)